src/docs/sdk/philosophy.mdx

```diff
@@ -57,3 +57,7 @@ Please check <Link to="/sdk/data-handling">Data Handling</Link> for more detail.
 ## Don’t forget the big picture
 
 Being correct is important but even more important is doing the right thing. Yes, we are building libraries and with that comes much responsibility. But in the end, we build a product that should solve a problem users have in their application. This principle relates to many other things mentioned on this page - but most importantly, it should act as a reminder that we only succeed if we manage to solve a problem end-to-end.
+
+## Write less code
+
+Only do in SDKs what can only be done in SDKs. Data quality is one of the most important things. Without it, we can't build a great product experience. The purpose of the SDK is to instrument the user's application, collect context around the event type we want to capture, and send it to Sentry. Any extensive business logic that does changes to the data should be avoided; the data should be sent as raw as possible. Business logic in SDKs is complex, hard to maintain and even harder to change and becomes baked into the data forever. We have way more flexibility on the server. 
```

Comment:
```suggestion
The role of an SDK is to instrument the user's application, collect context around the event types we want to capture, and send event data with useful context to Sentry. We should avoid extensive business logic within the SDK that transforms this data as it can get complex, hard to maintain and makes permanent changes to the wire format we send events. We should aim to have data collected by the SDK be as raw as possible. We have more flexibility and control on the server to manipulate the data more effectively.
```